src/ui/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,156 @@
+
+    tabList: ListModel {
+        id: tabButtonList
+        ListElement {
+            tabLabelStringId: "TipsAndTricksTabTitleAll"
+            tabButtonId: "tabAll"
+        }
+        ListElement {
+            tabLabelStringId:"TipsAndTricksTabTitleTutorials"
+            tabButtonId: "tabTutorials"
+        }
+        ListElement {
+            tabLabelStringId:"TipsAndTricksTabTitleTips"
+            tabButtonId: "tabTips"
+        }
+    }
+
+    stackContent: [
+        // All
+        VPNFlickable {
+            flickContentHeight: layoutAll.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+            interactive: flickContentHeight > height
+
+            ObjectModel {
+                id: tipsAndTricksSections
+
+                Item { 
+                    property string title: VPNl18n.TipsAndTricksQuickTipsTitle
+                    property string description: VPNl18n.TipsAndTricksQuickTipsDescription
+                    property var filter: (addon) => !addon.advanced && !addon.highlighted
+                }
+
+                Item { 
+                    property string title: VPNl18n.TipsAndTricksAdvancedTipsTitle
+                    property string description: VPNl18n.TipsAndTricksAdvancedTipsDescription
+                    property var filter: (addon) => addon.advanced && !addon.highlighted
+                }
+            }
+
+            ColumnLayout {
+                id: layoutAll
+
+                anchors.fill: parent
+                anchors.topMargin: VPNTheme.theme.vSpacing
+                anchors.bottomMargin: VPNTheme.theme.vSpacing
+                anchors.leftMargin: VPNTheme.theme.windowMargin
+                anchors.rightMargin: VPNTheme.theme.windowMargin
+
+                VPNTutorialList {
+                    parentWidth: layoutAll.width
+                    customTutorialFilter: (addon) => addon.highlighted
+                }
+
+                Repeater {
+                    model: tipsAndTricksSections.count
+
+                    Column {
+                        property var section: tipsAndTricksSections.get(index)
+
+                        visible: guidesList.count || tutorialsList.count
+
+                        Layout.alignment: Qt.AlignTop
+                        Layout.topMargin: 32
+                        spacing: VPNTheme.theme.vSpacingSmall
+
+                        VPNBoldLabel {
+                            text: section.title
+
+                            Accessible.role: Accessible.StaticText
+                            Accessible.name: text
+                        }
+
+                        VPNTextBlock {
+                            text: section.description
+
+                            Accessible.role: Accessible.StaticText
+                            Accessible.name: text
+                        }
+
+                        VPNGuideList {
+                            id: guidesList
+                            parentWidth: layoutAll.width
```

Comment:
You probably don't need this `parentWidth` property, maybe try setting horizontal anchors, and just basing the number of columns off of the internal width (i.e. `columns: width < VPNTheme.theme.tabletMinimumWidth ? 2 : 3`)

Reply:
This will likely depend on [this change](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4221/files#r953285426)